Exiled.API/Features/Pickups/Scp330Pickup.cs

```diff
@@ -0,0 +1,62 @@
+// -----------------------------------------------------------------------
+// <copyright file="Scp330Pickup.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features.Pickups
+{
+    using System.Collections.Generic;
+
+    using Exiled.API.Features.Items;
+
+    using InventorySystem.Items.Usables.Scp330;
+
+    using BaseScp330 = InventorySystem.Items.Usables.Scp330.Scp330Pickup;
+
+    /// <summary>
+    /// A wrapper class for SCP-330 bags.
+    /// </summary>
+    public class Scp330Pickup : Pickup
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Scp330Pickup"/> class.
+        /// </summary>
+        /// <param name="itemBase">The base <see cref="BaseScp330"/> class.</param>
+        public Scp330Pickup(BaseScp330 itemBase)
+            : base(itemBase)
+        {
+            Base = itemBase;
+        }
+
+        /// <summary>
+        /// Gets the <see cref="BaseScp330"/> that this class is encapsulating.
+        /// </summary>
+        public new BaseScp330 Base { get; }
+
+        /// <summary>
+        /// Gets or sets the exposed type.
```

Comment:
`Gets or sets the exposed <see cref="CandyKindID">`

Reply:
Done